Makefile

```
@@ -0,0 +1,149 @@
+
+.PHONY: cover-kdtree
+cover-kdtree: ## Print coverage details for kdtree.go only
+	@$(GO) tool cover -func=$(COVEROUT) | grep 'kdtree.go' || true
+
+.PHONY: coverhtml
+coverhtml: cover ## Generate HTML coverage report at $(COVERHTML)
+	@$(GO) tool cover -html=$(COVEROUT) -o $(COVERHTML)
+	@echo "Wrote $(COVERHTML)"
+
+.PHONY: fuzz
+fuzz: ## Run Go fuzz tests for $(FUZZTIME)
```

checkmake (0.2.2)

[warning] Target body for "fuzz" exceeds allowed length of 5 (13).

(maxbodylength)